script/ghci-flags

```diff
@@ -37,6 +37,7 @@ function flags {
   echo "-isemantic-analysis/src"
   echo "-isemantic-ast/src"
   echo "-isemantic-core/src"
+  echo "-isemantic-diff/src"
```

Comment:
❤️

I’ve been meaning to add a note to the `cabal.project{,.ci}` files to remember to do this, so thank you for remembering anyway 😊